server/server.go

```diff
@@ -90,16 +92,18 @@ func main() {
 	}
 
 	err := sentry.Init(sentry.ClientOptions{
-		Dsn: 			sentryConf.Dsn,
-		Debug: 			sentryConf.Debug,
+		Dsn:              sentryConf.Dsn,
+		Debug:            sentryConf.Debug,
+		AttachStacktrace: sentryConf.AttachStacktrace,
 	})
 
 	if err != nil {
 		log.Fatalf("sentry.Init: %s", err)
 	}
 
 	defer sentry.Flush(2 * time.Second)
-	
+	defer sentry.Recover()
```

Comment:
This is for handling panics in the main routine: 
https://sentry.io/organizations/dapperlabs/issues/3453939358/?project=6398442&query=&statsPeriod=14d